Jenkinsfile

```diff
@@ -7,22 +7,9 @@ node {
   govuk.setEnvar("TEST_DATABASE_URL", "postgresql://postgres@127.0.0.1:54313/publishing-api-test")
 
   govuk.buildProject(
-    extraParameters: [
-      stringParam(
-        name: "CONTENT_STORE_BRANCH",
```

Comment:
We might still need this bit? Unless I've misundertstood

Reply:
yea whoops that was my bad just fixing!